pygerduty/__init__.py

```diff
@@ -505,9 +505,21 @@ def __init__(self, *args, **kwargs):
         Container.__init__(self, *args, **kwargs)
         self.notification_rules = NotificationRules(self.pagerduty, self)
         self.contact_methods = ContactMethods(self.pagerduty, self)
+        self.schedules = UserSchedules(self.pagerduty, self)
+        self.escalation_policies = UserEscalationPolicies(self.pagerduty, self)
         self.log_entries = LogEntries(self.pagerduty, self)
 
 
+class UserSchedules(Collection):
```

Comment:
To be more clear, adding this

```python
        self.schedules = Schedules(self.pagerduty, self)
        self.escalation_policies = EscalationPolicies(self.pagerduty, self)
```

to the `User` container seemed to work without needing the two new collection classes you added.